.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 public
+.stylelintrc
```

Comment:
Remove line 3 and add/commit the .stylelint file. This is a file you'd want to keep in the repo, that way anyone else that has the project locally can adhere to the same linting rules.